test/integration/default/squid_spec.rb

```diff
@@ -5,6 +5,12 @@
   it { should be_listening }
 end
 
+cache_dir = os[:linux?] ? '/var/spool/squid' : '/var/squid/cache'
```

Comment:
Did you meant to have linux? inside the os hash?

Reply:
Can do `os.linux?` if that's more idiomatic. Both work.

Reply:
Switched to `os.linux?` for better readability.